<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>eepromwidget</class>
 <widget class="QMainWindow" name="eepromwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройка EEPROM</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="gbFlashData">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="sul_flash_layout">
         <item>
          <widget class="QTableWidget" name="eepromTable">
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>43</number>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>24</number>
           </attribute>
           <row>
            <property name="text">
             <string>1</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>2</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>3</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>4</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>5</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>6</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>7</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>8</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>1</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>2</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>3</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>4</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>5</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>6</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>7</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>8</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>9</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>10</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>11</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>12</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>13</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>14</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>15</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>16</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_21">
      <property name="title">
       <string>Управление конфигурацией FLASH</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_5">
         <property name="spacing">
          <number>6</number>
         </property>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_30">
           <property name="text">
            <string>Команды</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="2" colspan="2">
          <widget class="QLabel" name="label_24">
           <property name="text">
            <string>Информация</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_25">
           <property name="text">
            <string>CRC абонентов:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>Сегмент</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="4" colspan="2">
          <widget class="QLabel" name="label_29">
           <property name="text">
            <string>Навигация</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QSpinBox" name="spbFlashAbonentNumber">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>0</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLabel" name="lblFileCRC">
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>0x0000</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLabel" name="label_33">
           <property name="text">
            <string>Абонент</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="lblHwCRC">
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>0x0000</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="btnFlashErase">
           <property name="text">
            <string>Стирание</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="btnFlashSave">
           <property name="text">
            <string>Сохранить</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="btnFlashFileOpen">
           <property name="text">
            <string>Открыть</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="btnFlashLoad">
           <property name="text">
            <string>Загрузка</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_27">
           <property name="text">
            <string>CRC файла:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="5">
          <widget class="QComboBox" name="cbFlashSegmentName"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>748</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
